core/sql/optimizer/RelGrby.h

```diff
@@ -552,7 +560,9 @@ class GroupByAgg : public RelExpr
   // ---------------------------------------------------------------------
   ItemExpr    * groupExprTree_;
   ValueIdSet  groupExpr_;
-
+  
+  ItemExpr  * extraGrpOrderby_;
+  ValueIdList  extraOrderExpr_;
```

Comment:
It is possible to combine, but not very simple. Since there are different purpose to add extra order, and in some cases, for example display, it needs to tell which orderby is used for rollup and which is for group_concat, it is possible but will complicated the code. So I prefer to keep it as for now.